variables.tf

```diff
@@ -751,36 +757,38 @@ variable "clusters" {
   description = "A list describing clusters workloads to create"
   type = list(
     object({
-      name               = string           # Name of Cluster
-      vpc_name           = string           # Name of VPC
-      subnet_names       = list(string)     # List of vpc subnets for cluster
-      workers_per_subnet = number           # Worker nodes per subnet.
-      machine_type       = string           # Worker node flavor
-      kube_type          = string           # iks or openshift
-      kube_version       = optional(string) # Can be a version from `ibmcloud ks versions` or `latest`
-      entitlement        = optional(string) # entitlement option for openshift
-      pod_subnet         = optional(string) # Portable subnet for pods
-      service_subnet     = optional(string) # Portable subnet for services
-      resource_group     = string           # Resource Group used for cluster
-      cos_name           = optional(string) # Name of COS instance Required only for OpenShift clusters
-      update_all_workers = optional(bool)   # If true force workers to update
+      name                    = string           # Name of Cluster
```

Comment:
Changes to the variables names will break the SLZ GUI.  The new optional ones are fine but the existing ones being changed will break it

@vburckhardt

Reply:
@argeiger , @vburckhardt 

I have reverted the changes to the variable names in the `worker_pools` list within the `clusters` variable. Regarding the variable `ocp_version` should I revert it to `kube_version`? Because we are not deploying k8s cluster anymore using the current code, so I guess `ocp_version` is the more suitable variable name.

Reply:
I have added the `kube_version` back, but it doesn't serve any purpose. The `ocp_version` variable is being used to determine the ocp version.